packages/esm-patient-chart-app/src/patient-flags/patient-flags-list.component.tsx

```diff
@@ -61,17 +61,17 @@ const PatientFlagsList: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeW
       showToast({
         critical: true,
         kind: 'success',
-        description: t('flagEnabled', 'Patient flag successfully enabled'),
-        title: t('enablingFlag', 'Enabling flag'),
+        description: t('flagEnabledSuccessfully', 'Flag successfully enabled'),
+        title: t('enabledFlag', 'Enabled flag'),
       });
 
       setEnabling(false);
     } else {
       showNotification({
         critical: true,
         kind: 'error',
-        description: t('errorEnabling', 'There was a problem enabling the flag'),
-        title: t('enablingFlag', 'Enabling flag'),
+        description: t('flagEnableError', 'Error enabling flag'),
+        title: t('flagEnabled', 'flag enabled'),
```

Comment:
I believe the description should be the title and no description needs to be there

Reply:
Title says enabled but the description says error occured